godip.go

```diff
@@ -478,6 +478,8 @@ type Validator interface {
 	GetProfile() (map[string]time.Duration, map[string]int)
 
 	MemoizeProvSlice(string, func() []Province) []Province
+
+	Flags() map[Flag]bool
```

Comment:
This is ok, but currently the only flags are mutually exclusive.

Reply:
True - but I figured this might not be the only thing we wanted to add, and the concept of flags is already there in build orders (which use the same two mutually exclusive ones, by the way - thanks to you :stuck_out_tongue: )